src/analytics/task/query.ts

```diff
@@ -10,3 +10,74 @@ export const GET_TASK_EVENT_DATA = gql`
     }
   }
 `;
+
+export const GET_ANNOTATION_EVENT_DATA = gql`
```

Comment:
Are all these fields necessary for the data needed for the annotation analytics?

Reply:
I spoke to Maria and we figure out which fields can be removed. Thanks!